sample/build.gradle

```diff
@@ -24,8 +74,8 @@ android {
         applicationId "io.ona.kujaku.sample"
         minSdkVersion 16
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 1
-        versionName "1.0"
+        versionCode getMasterCommitCount()
+        versionName getVersionName()
```

Comment:
How would one test if this works?

Reply:
Run the app on your device and check the version name, it should be somewhere around `v0.1.4` and some other identifiers